run.sh

```diff
@@ -70,17 +68,10 @@ printf "\t%s\n" \
 # wp-config.php
 # -------------
 printf "=> Generating wp.config.php file... "
-if [ ! -f /app/wp-config.php ]; then
-  sudo -u www-data wp core config >/dev/null 2>&1 || \
-    ERROR $LINENO "Could not generate wp-config.php file"
-  printf "Done!\n"
-else
-  printf "Already exists!\n"
-fi
-
-# Enable or Disable debug mode
-sed -i "s/define('WP_DEBUG', \(true\|false\))/define('WP_DEBUG', ${WP_DEBUG,,})/g" /app/wp-config.php
-
+rm -f /app/wp-config.php
```

Comment:
@karellm Thoughts on this (and line 20)?

This way, `wp-cli.yml` and `wp-config.php` are always overwritten (and should always contain the right options) on each build.